SoftFox/SoftFox/Level.h

```diff
@@ -33,5 +37,6 @@ class Level
 	bool* levelData;
 	Coordinates startPosition;
 	Coordinates villianPosition;
+	Coordinates wallPosition;
```

Comment:
Presumably your level contains more than one wall, so does it make sense to store a single `wallPosition`?